### ISSUE TITLE
WebHook Issue

### ISSUE THREAD

**Issue by DarthMaus** [Tenclea/YANG#150]
I fixed the problem with the spaces in the config, and set `status_update_interval: 0`, so it wouldn't send any updates, but it still does:
[IMAGE](https://media.discordapp.net/attachments/1141302966456946848/1148142370995580989/image.png?width=530&height=662)
Why does it do that / how do I fix his?

**Comment by DarthMaus** [comment]
My whole config:
```
# The length of the codes to generate, can be one of 'short', 'long' or 'both'
# Short is 16 characters, long is 24, and both will randomly choose between 16 and 24 each time
code_length: short

# Auto redeems valid nitro codes
auto_redeem:
  # enable auto_redeem or not (true / false)
  enabled: false
  # The token to redeem the codes with
  token: "PUT_YOUR_TOKEN_HERE"

# Proxy-related settings
proxies:
  # Write working proxies to a file (true / false)
  save_working: true

  # Validate proxies before passing them to the checker (true / false)
  enable_checker: true

  # Download fresh proxies from the web (true / false)
  enable_scrapper: true

  # The maximum amount of proxies to download (if enabled), use '0' for no maximum.
  max_proxies_download: 5000

# The amount of codes to check at the same time
# The higher, the faster, but it may slow the generator down if it is set too high
threads: 250

# Receive webhook messages on start and when a valid code is found
webhook:
  # enable webhook messages or not (true / false)
  enabled: true

  # The webhook url
  url: https://discord.com/api/webhooks/******

  # Webhook notifications settings
  notifications:
    # Send a notification when the generator starts (true / false)
    boot: false

    # Send a notification when a valid code is found (true / false)
    valid_code: true

  # How often to send status updates to the webhook in seconds (use '0' for never)
  status_update_interval: 0

# Print additional information in the console (true / false)
debug_mode: false
```

**Comment by Tenclea** [comment]
Fixed in v3.3.4 :)
